docs/core/config/auth/databases/ldap.md

```diff
@@ -510,29 +520,29 @@ olcAccess: {1}to *
     by dn="cn=authuser,dc=test,dc=dovecot,dc=net" read
     by * auth
 ```
+:::
 
 2. Run `ldapmodify` to apply the change.
 
-```sh
-ldapmodify -Q -Y EXTERNAL -H ldapi:/// -f doveauth_access.ldif
+```console
+$ ldapmodify -Q -Y EXTERNAL -H ldapi:/// -f doveauth_access.ldif
```

Comment:
also "sh" - although this seems to be deleted by a later change, or at least I don't see it anymore.

Reply:
Ah, it gets moved to data/settings.js and is escaped there so my grep didn't find it.